concordium-contracts-common/src/lib.rs

```diff
@@ -83,6 +83,16 @@
 #[cfg(not(feature = "std"))]
 extern crate alloc;
 
+/// Re-export.
+#[cfg(feature = "std")]
+pub use std::{string::String, vec::Vec};
```

Comment:
I don't think we should be re-exporting things in contracts-common.

We do it in concordium-std to provide an easy experience for users of that library, but contracts-common is different.

If you look at the comments I have below, this is not necessary.

Reply:
I don't re-export them anymore but I still import them here since `SchemaType` needs it.